pages/Notifications.js

```diff
@@ -1,10 +1,113 @@
 import React from 'react';
 
-export default () => (
-  <main>
-    <section>
-      <h2>Case Headers</h2>
-      <p>TBC</p>
-    </section>
-  </main>
-);
+export default () => {
+  return (
+    <main>
+      <section>
+        <h2>Alerts and Toasts</h2>
+        <h4>Overview</h4>
+        <p>
+          Alerts and toasts are used to communicate a message with a severity
+          attached to it. They grab the user’s attention to provide critical
+          information needed in context.
+        </p>
+        <h4>Validation Messages</h4>
+        <p>
+          Validation messages are error-handling messages related to user’s
+          inputs and they communicate whether the data just entered was
+          incomplete or incorrect. Message validation can be performed either in
+          real-time or after the user triggers the action. The notification
+          should inform in plain and simple language that a non-technical user
+          understands, so that it does not add to confusion or demoralises the
+          user.
+        </p>
+        <h4>Alerts</h4>
+        <p>
+          Alerts announce an event that has some significance for the users,
+          they triggered by the system and not by user’s immediate actions. It
+          is recommended to use compact-size and concise language as you share
+          an alert.
+        </p>
+
+        <b>App level alert</b>
```

Comment:
Don't use `<b>` to make things bold, use `<strong>` instead.

Could this also not be h5? as its a heading?

Reply:
Initially I used h5, the text looked a bit dispersed. I can still revert it without problems, do I proceed?

Reply:
i'm not sure what you mean. It might look better though if everything is in a section. You might be able to use h3/4 instead then

Reply:
![image](https://user-images.githubusercontent.com/57677593/71908338-b4066f80-3165-11ea-931f-61139bd1e0c4.png)
It would look like this